docs/usage/self-hosted-configuration.md

```diff
@@ -665,6 +670,27 @@ By default, Renovate logs and displays a warning when the `GITHUB_COM_TOKEN` is
 By setting `githubTokenWarn` to `false`, Renovate suppresses these warnings on Pull Requests, etc.
 Disabling the warning is helpful for self-hosted environments that can't access the `github.com` domain, because the warning is useless in these environments.
 
+## gitlabAutoMergeableCheckAttempts
+
+If set to an positive integer, Renovate will use this as the number of attempts to check if a merge request on GitLab is mergeable before trying to automerge.
+The formula for the delay between attempts is `gitlabAutoMergeableCheckAttempts * attempt * attempt` milliseconds.
+
+Default value: `5` (attempts results in max. 13.75 seconds timeout).
+
+## gitlabBranchStatusDelay
+
+Adjust default time (in milliseconds) given to GitLab to create pipelines for a commit pushed by Renovate.
+
+Can be useful for slow-running, self-hosted GitLab instances that don't react fast enough for the default delay to help.
+
+Default value: `1000` (milliseconds).
+
+## gitlabMergeRequestDelay
+
+If set, Renovate will use this as a delay to proceed with an automerge.
```

Comment:
```suggestion
If set, Renovate will wait this extra time before an automerge.
```

Maybe something like this is easier to read.

Reply:
I think delay is more commonly used.